File: submissions/kiprobalasibol/C1.js
function f(x,c){
var l=require('fs').readFileSync(x).toString().split("\n");
var s = 0;
for (var i = 0; i < l.length; i++) {
	if(l[i].lastIndexOf(c)>-1){
		x=l.length-i-1;
		s = l.length-l[i].lastIndexOf(c) - 1;
	}
}
process.stdout.write(x+','+s);
}

File: submissions/kiprobalasibol/C1.cs
static void f(string x, char c)
{
    int r = 0, s = 0;
    var l = File.ReadAllLines(x);

    for (int i = 0; i < l.Count(); i++)
    {
        if (l[i].LastIndexOf(c) > -1)
        {
            r = l.Count() - i - 1;
            s = l.Count() - l[i].LastIndexOf(c) - 1;
        }
    }

    Console.Write("{0},{1}", r, s);
}
        